asm-to-pil/Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "powdr-asm-to-pil"
-version = "0.1.0"
-edition = "2021"
+description = "powdr asm to PIL compiler"
```

Comment:
This crate transforms vm machines to constrained machines, i.e. translates the asm instructions to constraints. The result is PIL in the same way as "fully analyzed asm" is pil.